docs/docs-content/release-notes/release-notes.md

```diff
@@ -11,376 +11,177 @@ tags: ["release-notes"]
-the following sections for a complete list of features, improvements, and known issues.
+Sit et beatae expedita rerum odit ullam quos id non voluptas quae et nisi. Est commodi excepturi accusamus sed incidunt
+doloribus. Praesentium qui accusantium amet. Eaque voluptates impedit sint amet omnis delectus velit tempora eos maxime
+laudantium corporis iure. Nesciunt nihil et asperiores dolore suscipit dolor architecto. Fugit distinctio iste maxime
+saepe ut fugit ea ut architecto quae consequatur.
 
 ### Security Notices
 
-- Kubernetes version 1.27.9 is deprecated due to a security vulnerability. We recommend upgrading to a newer version of
-  Kubernetes, such as 1.27.11, to avoid issues.
-
 - Review the [Security Bulletins](../security-bulletins/security-bulletins.md) page for the latest security advisories.
 
 ### Palette
 
-#### Features
+#### Breaking Changes
 
-<!-- prettier-ignore -->
-- <TpBadge /> Cluster Profile variables, a new feature that allows you to define variables in a cluster profile. This
-  feature is in Tech Preview and is available only for Edge clusters using Local UI. Profile variables allow you to define variable
-  types, apply validation, and narrow the scope of variables to a cluster profile. 
-  Check out [Cluster Profile Variables](../profiles/cluster-profiles/create-cluster-profiles/define-profile-variables.md) to learn more about
-  profile variables.
-
-- MAAS clusters using Palette eXtended Kubernetes (PXK) now support the ability to specify a custom MAAS API endpoint
-  URL and port during cluster creation. This feature allows you to use a custom DNS server or Virtual IP (VIP) that is
-  not resolvable outside of the MAAS network. Refer to the [PXK](../integrations/kubernetes.md#custom-maas-endpoint)
-  documentation for more details.
-
-- Support for [Konvoy](../integrations/konvoy.md) is now available in Palette. You can create a custom image using the
-  Konvoy image builder project and use it to deploy a Konvoy cluster. Check out the
-  [Red Hat Linux Enterprise and Konvoy](../byoos/usecases/vmware/konvoy.md) guide to learn how to create a custom image
-  and deploy a Konvoy cluster.
-
-- Multiple system administrators can now be added to the self-hosted Palette system console to help manage and maintain
-  the Palette instance. The feature helps organizations embrace the separation of duties by delegating different
-  responsibilities to system administrators. Refer to the
-  [System Administrators](../enterprise-version/system-management/account-management/account-management.md#system-administrators)
-  page to learn more about system administrators.
+- Google Cloud Platform (GCP) does not support the ability for users to specify a patch version for the Kubernetes
+  version used in GKE clusters. In this release, Palette aligns with GCP's behavior and removes the ability to specify a
+  patch version when creating a cluster profile for GKE. Only the major and minor versions are available for selection.
```

Comment:
GKE allows to select a specific patch version, but most of the time there's only one available for each minor version. However, they automatically deprecate and remove patch version without warning when they release a new one. To cope with that, we implemented the same system as for AKS and EKS, showing only major+minor.

I'm still searching for this blog article explaining that, and I also asked engineering to search it.

Reply:
Updated the language